daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/ExpressionEvaluatingParsers.scala

```diff
@@ -67,6 +83,82 @@ class IVCParser(expr: CompiledExpression[AnyRef], e: ElementRuntimeData)
     }
 }
 
+/*
+ * Run a parser for an element that does not occur in the infoset
+ * Prior to running, a temporary element (of the type expected by the parser) will be created in the infoset,
+ * After running, the infoset will be reverted to its original state, but any other side effect of parsing will remain
+ *
+ * Additionally, the dataValue of the element the parser parsed will be returned
+ */
+trait withDetachedParser {
```

Comment:
Captial trait name. Also, should this be WithQuasiParser? Detached was changed to Quasi in the grammar, should probably be changed here for consistency.

Reply:
We changed DetachedElement -> QuasiElement in the grammar. The idea being that the element in question is not a "real" element, but just exists to satisfy the requirements of certain existing code.

In contrast, a detachedParser strikes me as a "real" parser, so the quasi adjective does not make sense to me here.

Reply:
Fair, makes sense to me.